client/src/app/pages/issues/affected-applications/affected-applications.tsx

```diff
@@ -0,0 +1,237 @@
+import * as React from "react";
+import {
+  Button,
+  DescriptionList,
+  DescriptionListDescription,
+  DescriptionListGroup,
+  DescriptionListTerm,
+  Flex,
+  FlexItem,
+  Label,
+  PageSection,
+  PageSectionVariants,
+  Text,
+  TextContent,
+  TextVariants,
+  Toolbar,
+  ToolbarContent,
+  ToolbarItem,
+  Tooltip,
+} from "@patternfly/react-core";
+import { useTranslation } from "react-i18next";
+import { AppPlaceholder, ConditionalRender } from "@app/shared/components";
+import {
+  FilterToolbar,
+  FilterType,
+} from "@app/shared/components/FilterToolbar";
+import {
+  ExpandableRowContent,
+  TableComposable,
+  Tbody,
+  Td,
+  Th,
+  Thead,
+  Tr,
+} from "@patternfly/react-table";
+import {
+  useTableControlUrlParams,
+  useTableControlProps,
+  getHubRequestParams,
+  useFilterState,
+} from "@app/shared/hooks/table-controls";
+import { useCompoundExpansionState } from "@app/shared/hooks/useCompoundExpansionState";
+import { SimplePagination } from "@app/shared/components/simple-pagination";
+import {
+  ConditionalTableBody,
+  TableHeaderContentWithControls,
+  TableRowContentWithControls,
+} from "@app/shared/components/table-controls";
+import { useFetchIssues } from "@app/queries/issues";
+import spacing from "@patternfly/react-styles/css/utilities/Spacing/spacing";
+import textStyles from "@patternfly/react-styles/css/utilities/Text/text";
+import { Link, useParams } from "react-router-dom";
+import { useLocalTableControls } from "@app/shared/hooks/table-controls";
+import { useFetchApplications } from "@app/queries/applications";
+import { Application } from "@app/api/models";
+interface IAffectedApplicationsPageParams {
+  ruleid: string;
+}
+export const AffectedApplications: React.FC = () => {
+  const { t } = useTranslation();
+  const { ruleid = null } = useParams<IAffectedApplicationsPageParams>();
```

Comment:
@mturley This works for now, but wondering if I should just somehow pass the ruleid from the parent composite issues table in another way.